src/renderers/materialized/materialized.datetime.control.tsx

```diff
@@ -0,0 +1,95 @@
+import { JSX } from '../JSX';
+import { withIncreasedRank } from '../../core/testers';
+import { Control, ControlProps, ControlState } from '../controls/Control';
+import {
+  assembleDateTime, mapStateToControlProps, registerStartupRenderer
+} from '../renderer.util';
+import { datetimeControlTester } from '../controls/datetime.control';
+import { connect } from '../../common/binding';
+declare let $;
+export class DateTimeControl extends Control<ControlProps, ControlState> {
+
+  componentDidMount() {
+    $('.datepicker').pickadate({
```

Comment:
We discussed the problems you encounter with date, time and datetime and we would like to try out the use of a react based materialize library like: https://material-ui-next.com/ (http://www.material-ui.com/) .
Could you implement the datetime using this library in version 1.0.0-alpha and probably one of the mentioned picker libraries in the docu and then create a new PR?

Reply:
To avoid any confusion, the current version of material-ui at the time of writing is v1.0.0-beta.22.